template/.drone.yml

```diff
@@ -7,21 +7,39 @@
 
 matrix:
   include:
+    # builds every branch but only deploys if branch is "dev"
     - BUILD_ENV: dev
-      REACT_APP_API_BASE_URL: https://your-public-url-dev.example.com
-      REACT_APP_DEPLOYMENT_ENV: dev:optimized
```

Comment:
REACT_APP_DEPLOYMENT_ENV war dazu da, um das build/deployment env in der App anzeigen zu können. War in manchen Projekten nützlich für Code, der z.B. nicht in production laufen soll, oder nur auf dev.

Reply:
Will add it again.